<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchPatientDialog</class>
 <widget class="QDialog" name="SearchPatientDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>201</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Пациенты - Поиск</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayoutDataForm">
     <item row="2" column="0">
      <widget class="QRadioButton" name="radioButtonAge">
       <property name="text">
        <string>Возраст</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lineEditPhone">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QRadioButton" name="radioButtonFullname">
       <property name="text">
        <string>ФИО</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="spinBoxAge">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimum">
        <number>18</number>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEditFullname">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QRadioButton" name="radioButtonAddress">
       <property name="text">
        <string>Адрес</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QRadioButton" name="radioButtonPhone">
       <property name="text">
        <string>Номер телефона</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="0">
        <widget class="QLabel" name="labelDistrict">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Район</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="labelRegion">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Регион</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEditRegion">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="comboBoxDistrict">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButtonSearch">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Искать</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCancel">
       <property name="text">
        <string>Отмена</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
